starfyre/parser.py

```diff
@@ -159,33 +159,32 @@ def handle_endtag(self, tag):
         if tag not in self.generic_tags and tag not in self.components:             
             raise UnknownTagError(f'Unknown tag: "{tag}". Please review line {self.lineno} in your "{self.component_name}" component in the pyml code.')
         
-        endtag_node = self.stack.pop()  
+        endtag_node = self.stack.pop()
         self.current_depth -= 1
 
         if endtag_node.tag != "style" and endtag_node.tag != "script":
+            
+            if endtag_node.original_name != endtag_node.tag: #We need to check if the endtag_node is a custom tag
```

Comment:
Niceee. There is one final suggestion and everything should be hopefully good to go. I will complete my review by tonight. 😄